samples/MusicStore/Program.cs

```diff
@@ -15,8 +16,10 @@ public static void Main(string[] args)
                 .AddEnvironmentVariables(prefix: "ASPNETCORE_")
                 .Build();
 
+            var currentAssemblyLocation = new FileInfo(typeof(Program).GetTypeInfo().Assembly.Location);
+
             var builder = new WebHostBuilder()
-                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseContentRoot(currentAssemblyLocation.Directory.FullName)
```

Comment:
True this breaks VS experience. When the NanoServer tests get executed they currently do not run from the same directory as where the published output is (another reason why GetCurrentDirectory usage is not always correct). I will try fixing NanoServer tests instead. 

BTW, we used to have `copyToOutput` in project.json world, do we have something equivalent in msbuild build too?